internal/wire/wire_test.go

```diff
@@ -118,6 +119,19 @@ func fuzzMessages(f *testing.F, testCases []testCase) {
 		f.Add(tc.expectedB)
 	}
 
+	records, err := loadRecords(filepath.Join("..", "..", "records"))
+	if err != nil {
+		f.Fatal(err)
+	}
```

Comment:
```suggestion
	require.NoError(t, err)
```

I think we should not mix stdlib-style and testify-style assertions